<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(211, 215, 207);</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonClose">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButtonClose</zorder>
   <zorder>textEdit</zorder>
   <zorder>pushButtonUpdate</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
